### ISSUE TITLE
Fix the assess() function

### ISSUE THREAD

**Issue by rimorob** [rimorob/netres#3]
Plot AUCs and PR-AUCs not as dots but on the same AUC plot with error bars, using either/both of ROCR and PROC

**Comment by fkgruber** [comment]
we want:
- pvalues
- lift curves

**Comment by fkgruber** [comment]
Hi Boris @rimorob 
I have a first version of the assess plotting.  It plots what you have now which is auc vs iteration and also saves the roc curves at each iteration as:

![image](https://user-images.githubusercontent.com/4311763/226199702-0c4e294a-3de0-4f81-bd56-727b766deba1.png)

One question I have is regarding the p-value. THe ROCR test function allows to compare 2 roc curves. Were you thinking to compare the ROC of the true vs each iteration or each iteration with estimated latent variable vs the roc curve if the 1st iteration (ignoring latent variables)?

**Comment by rimorob** [comment]
Yes, I think AUC vs the true network (minus the edges crossing into the latent space) would be used as the basis for comparison

Looks nice!

**Comment by rimorob** [comment]
Fred, is this fully finished?

**Comment by fkgruber** [comment]
I think I'm missing the p-value

**Comment by fkgruber** [comment]
> Yes, I think AUC vs the true network (minus the edges crossing into the latent space) would be used as the basis for comparison
> 
> Looks nice!

Actually roc.test works by providing 2 roc objects which mean we need 2 estimated network vs the true.
one option comparing the network from having the true latent variable in vs the last estimated network. Perhaps we should learn this oracle network by default to use as reference at the end

**Comment by fkgruber** [comment]
I will let assess have an argument with the oracle performance so we can show pvalues
